net.worldofpadman.WoP.metainfo.xml

```diff
@@ -7,19 +7,25 @@
     <id>worldofpadman.desktop</id>
   </provides>
   <name>World of PADMAN</name>
-  <summary>Freeware Fun Shooter</summary>
+  <summary>World of PADMAN is an incredibly carefully designed and colorful freeware fun shooter for young and young-at-heart people.</summary>
```

Comment:
```suggestion
  <summary>Incredibly carefully designed and colorful freeware fun shooter for young and young-at-heart people</summary>
```

The summary is too long, should not contain the dot at the end and does not have to contain "World of PADMAN".

Reply:
It looks like that for some reason I am not able to commit the suggestion. It is up to you, @kai-li-wop. :-)

I will merge the PR after the summary is fixed.

Reply:
Where can I see the restrictions of length? Is there a best practice documented somwhere? I don't have time right now, but I will check the upcoming days.

Reply:
I am not sure (the [official spec documentation](https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-summary) states only that it should be "A short summary of what this component does."), however this is from my experience as a software maintainer. I worked on many things including GNOME Software (software store GUI for GNOME) over the years and have also written dozens of AppStream metadata and desktop files, so I know the best practices.

Anyway, this is where the summary gets displayed:

![summary](https://user-images.githubusercontent.com/2339930/210377841-6d6d7dc9-4fc0-40b4-ac5c-aed8c57e268a.png)

![summary2](https://user-images.githubusercontent.com/2339930/210377957-2a199e6f-d55f-43d0-a403-c9790674abc0.png)